src/plugins/load-occupancy-data.client.ts

```diff
@@ -1,8 +1,9 @@
+import type { Pinia } from "pinia";
 import { useSpacesStore } from "~/stores/spaces";
 import { asDictionary } from "../lib/collection-utils";
 
 export default defineNuxtPlugin(async (app) => {
-  const spacesStore = useSpacesStore(app.$pinia);
+  const spacesStore = useSpacesStore(app.$pinia as Pinia); //https://github.com/vuejs/pinia/issues/2071
```

Comment:
What do you think of removing this comment and putting it in a ticket?
It would make technical debt a bit clearer, we can use a GH perma link to this piece of code so it's clear where it's used.